lib/ansible/modules/cloud/vmware/vmware_cluster_vsan.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Joseph Callen <jcallen () csc.com>
+# Copyright: (c) 2018, Ansible Project
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_cluster_vsan
+short_description: Manage vSAN on VMware vSphere clusters
+description:
+    - This module can be used to manage vSAN on VMware vSphere clusters.
+    - All values and VMware object names are case sensitive.
+version_added: '2.9'
+author:
+- Joseph Callen (@jcpowermac)
+- Abhijeet Kasurde (@Akasurde)
+requirements:
+    - Tested on ESXi 5.5 and 6.5.
+    - PyVmomi installed.
+options:
+    cluster_name:
+      description:
+      - The name of the cluster to be managed.
+      type: str
+      required: yes
+    datacenter:
+      description:
+      - The name of the datacenter.
+      type: str
+      required: yes
+      aliases: [ datacenter_name ]
+    enable_vsan:
+      description:
+      - If set to C(yes) will enable vSAN when the cluster is created.
+      type: bool
+      default: 'no'
+    vsan_auto_claim_storage:
+      description:
+      - Determines whether the VSAN service is configured to automatically claim local storage
+        on VSAN-enabled hosts in the cluster.
+      type: bool
+      default: False
+      version_added: 2.8
```

Comment:
Same question about `version_added` throughout the new modules.

Reply:
Changed to 2.9.